documentation/src/main/asciidoc/topics/proc_configuring_cache_tracing.adoc

```diff
@@ -0,0 +1,71 @@
+[id='tracing_cache']
+= Configure tracing at cache level
+
+Once the tracing is link:#configuring-tracing_{context}[configured at server level],
+it will be automatically enabled by default for all caches.
+A cache configuration level of tracing allows on the other hand to enable or disable it at cache level and at runtime.
+
+[discrete]
+== Tracing categories
+
+Several categories are potentially traced:
+
+* Container. That are all the main cache operations, such as replace, put, clear, getForReplace, remove operations and size. With the exception of all the getting operation.
```

Comment:
What is `getFoReplace`? And so this doesn't include any retrieval operations? Should this not be something around write operations instead of just generic container then? And should we have trace get operations? Those should be the most frequent operation by far.

Reply:
Great observation. I think you're right since `getFoReplace` is a get operation and we don't trace get. Anyway this is the current behavior of the main and it is not related to this PR. So I would say that we can open an issue to address it. WDYT?

Reply:
Sure, I only brought it up since we this PR is documenting it now.

But I am still confused, what is `getForReplace`? I just want to make sure what operation you are referring to here.

Reply:
This is the function we trace => https://github.com/infinispan/infinispan/blob/ca06b77975971a83c304b13824ad0e8a1b3b0bca/server/hotrod/src/main/java/org/infinispan/server/hotrod/CacheRequestProcessor.java#L323

Reply:
So it is not `getForReplace` as I was saying. It is really just `replace`. If you look more closely at the method name it is just saying it is "handling" the get for a replace command. So in this case just remove the `getForReplace` as you already have replace in the sentence.